CMakeLists.txt

```diff
@@ -31,6 +31,14 @@ if (DOCS_ONLY)
   return()
 endif (DOCS_ONLY)
 
+# Sometimes `FindBlas` has trouble finding the threading library, but setting it
+# up this early resolves that issue. Since LIBXSMM is a Blas alternative we set
+# it up early too. Both Blas and Lapack are dependencies of Blaze, so both are
+# being set up before Blaze.
+include(SetupBlas)
```

Comment:
Why would setting it up earlier resolve the issue? Can you explain this? I've had the issue "resolved" by simply force pushing sometimes, and other times not. How were you able to figure out the order matters? And compare to what?

Reply:
Probably I just ended up on an unaffected GitHub-hosted VM when testing this "solution". I haven't been able to reproduce the issue, so dropped the commit. If the issue ever surfaces again I have a few more ideas to try, e.g. making sure Xcode is set up correctly with `xcode-select --install`.

Reply:
Opened https://github.com/sxs-collaboration/spectre/issues/2168 for reference if we encounter this issue again